addons/l10n_pe_pos/static/src/overrides/components/partner_list/partner_list.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+
+import {PartnerListScreen} from "@point_of_sale/app/screens/partner_list/partner_list";
+import {patch} from "@web/core/utils/patch";
+
+patch(PartnerListScreen.prototype, {
+    createPartner() {
+        super.createPartner(...arguments);
+        if (this.pos.isPeruvianCompany()) {
+            this.state.editModeProps.partner.city_id = [this.pos.cities[0].id, this.pos.cities[0].name];
+            this.state.editModeProps.partner.l10n_latam_identification_type_id = [
+                this.pos.l10n_latam_identification_types[0].id,
+                this.pos.l10n_latam_identification_types[0].name,
```

Comment:
Is a tuple guaranteed when the field is not set?

Reply:
Yes, installing Peruvian localization will install data for these fields.